test/sharness/t0700-remotepin.sh

```diff
@@ -319,6 +319,14 @@ test_remote_pins() {
 test_remote_pins ""
 
 test_kill_ipfs_daemon
+
+test_expect_success "'ipfs pin remote add' shows the warning message while offline" '
+  test_expect_code 0 ipfs pin remote add --service=test_pin_svc --enc=json $BASE_ARGS --name=name_a $HASH_A > actual &&
+  echo "Warning: the local node is offline and remote pinning may fail if there is no other provider" > expected &&
+  echo "{\"Status\":\"pinned\",\"Cid\":\"$HASH_A\",\"Name\":\"name_a\"}" >> expected &&
```

Comment:
We should instead grep for the specific `Warning: the local node is offline` line instead of expecting to match the entire full command JSON output. (There may be a more specialized internal tool than just `grep`, but I'm not familiar with all sharness internals.)

Reply:
Seems we use `grep` in this same test file so it should be ok.